contracts/axelar-gas-service/src/test.rs

```diff
@@ -22,14 +26,87 @@ fn setup_env<'a>() -> (Env, Address, Address, AxelarGasServiceClient<'a>) {
 
     let client = AxelarGasServiceClient::new(&env, &contract_id);
     let gas_collector: Address = Address::generate(&env);
-    client.initialize(&gas_collector);
```

Comment:
instead of removing this, duplicate part of the setup solely for test_initialize. All other tests will need to have an initialized contract anyways

Reply:
we could also standardize `setup_env` and `setup_and_initialize` as separate helpers